src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
+@import './utils/animation';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin #000 dotted;
```

Comment:
Ensure that the color values are consistent throughout the project. If you're using hex color values, stick to that format for all color declarations unless there's a specific reason to do otherwise.